### DISCUSSION TITLE
How was your experience?

### DISCUSSION THREAD

**Discussion by geckoking7** [geckoking7/kgt_idle#6] [Polls]
I need people to answer this question

**Comment by ExplodingHairyPotatoCat** [comment]
I like it, it has the same goofy ass vibe of ngu idle :)

**Reply by geckoking7** [reply]
You've played NGU?
Is it too similar to NGU idle?

**Reply by ExplodingHairyPotatoCat** [reply]
No, it's just the right amount similar. just enough to feel inspired, without being a direct copy/imitation.
